abiquo/client.py

```diff
@@ -94,6 +94,9 @@ def __getattr__(self, key):
             return self.__dict__[key]
         except KeyError as ex:
             return self._find_or_raise(key, ex)
+
+    def __setattr__(self, key, value):
+        self.__dict__[key] = value
```

Comment:
Hmmm not sure about how this behaves: the `__getattr__` checks the objct's dictionary but fallback to the json. Here we would be only modifying the internal dict but not the json itself, in case we want to *really* update the object?
Should we better assume that this object is *only* backed by the json, and remove the `__dict__` lookups from this method and the `__getattr__` one?